### ISSUE TITLE
Change the colour of text boxes, buttons etc. as the theme.

### ISSUE THREAD

**Issue by STEdirisuriya** [fossasia/meilix-generator#356]
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request

**Actual Behaviour**

Items in the window(text boxes, buttons...) are not similar to the theme.

**Expected Behaviour**

Change the colour of text boxes, buttons etc. as the theme.
I have attached some examples, (they are only examples)
![image](https://user-images.githubusercontent.com/33794760/49441377-4d898200-f7ec-11e8-88be-bbbe751f6bf9.png)
![image](https://user-images.githubusercontent.com/33794760/49441294-1b782000-f7ec-11e8-9a54-5d6a47919602.png)
![image](https://user-images.githubusercontent.com/33794760/49441302-203cd400-f7ec-11e8-9990-c11701875f7b.png)

**Would you like to work on the issue?**

Yes. I want to verify that could I work on this... @tabesin

**Comment by rpotter12** [comment]
@STEdirisuriya the web-app generator should be similar to other fossasia web-app generators

**Comment by tabesin** [comment]
We could of course ask UX designers to make a visual draft and implement it.The examples do not convince me so far but that is just for getting the idea as I understand.

**Comment by STEdirisuriya** [comment]
@tabesin 
> that is just for getting the idea as I understand. 

Yes, I think it should change, because there are a lot of issues with the theme. Some of them are as follows, 
- There is more free space between text boxes and the window, it needs to increase the width.
- The text that displayed when hovering is needed to change as a tooltip or something like that; when the button/text-box is not hovering there is huge space between them. 
- The blue colour of buttons is not suitable with the theme as I see.
- The footer should be fixed in the window.

Thank you! Am I close this issue?

**Comment by sarnava1** [comment]
@STEdirisuriya The points you have raised is good. You can open individual issues for each of them and then we will be happy to discuss it.

**Comment by STEdirisuriya** [comment]
@sarnava1 Oh, sure!! :smile:

**Comment by STEdirisuriya** [comment]
I'll open issues one by one.
